pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_versioning_a_product.adoc

```diff
@@ -15,15 +15,17 @@ A product must have at least one product version assigned to it.
 
 . Find the product and click the drop-down arrow>Product Details.
 
-. Enter the following information:
-  New Version:: This is the text that appears in {ProductShortName}, and the document preview.
-  URL Fragment:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
+. Provide the following information:
```

Comment:
Is there any plan to remove the conceptual content from this procedure? I wonder if this could be moved to a ref module or a glossary. I rewrote it to start each step with a verb.

Reply:
Look fine to me the way you rewrote it. I don't think that there's enough content here to stand on it's own in a separate module, and I also think it's appropriate to include this content here, as it's part of the procedure.